binding/tagcategory.go

```diff
@@ -23,29 +22,65 @@ func (h *TagCategory) Create(r *api.TagCategory) (err error) {
 func (h *TagCategory) Get(id uint) (r *api.TagCategory, err error) {
 	r = &api.TagCategory{}
 	path := Path(api.TagCategoryRoot).Inject(Params{api.ID: id})
-	err = h.Client.Get(path, r)
+	err = h.client.Get(path, r)
 	return
 }
 
 //
 // List TagCategories.
 func (h *TagCategory) List() (list []api.TagCategory, err error) {
 	list = []api.TagCategory{}
-	err = h.Client.Get(api.TagCategoriesRoot, &list)
+	err = h.client.Get(api.TagCategoriesRoot, &list)
 	return
 }
 
 //
 // Update a TagCategory.
 func (h *TagCategory) Update(r *api.TagCategory) (err error) {
 	path := Path(api.TagCategoryRoot).Inject(Params{api.ID: r.ID})
-	err = h.Client.Put(path, r)
+	err = h.client.Put(path, r)
 	return
 }
 
 //
 // Delete a TagCategory.
 func (h *TagCategory) Delete(id uint) (err error) {
-	err = h.Client.Delete(Path(api.TagCategoryRoot).Inject(Params{api.ID: id}))
+	err = h.client.Delete(Path(api.TagCategoryRoot).Inject(Params{api.ID: id}))
+	return
+}
+
```

Comment:
**hint**: Coped from addon/ and generally useful enough for all binding users to be included.
Find()
Ensure()

vvv